.mailmap

```diff
@@ -612,6 +612,7 @@ Gaurav Jain <gjain369@gmail.com> Gaurav Jain <72644006+gjain-7@users.noreply.git
 Gautam Menghani <gum3ng@protonmail.com> gautammenghani <gautammenghani14@gmail.com>
 Gautam Menghani <gum3ng@protonmail.com> gum3ng <78410304+gum3ng@users.noreply.github.com>
 Gautam Menghani <gum3ng@protonmail.com> gum3ng <gum3ng@protonmail.com>
+Geetika Vadali <geetika047btcsai21@igdtuw.ac.in> Geetika Vadali <geetika.vadali4@gmail.com>
```

Comment:
The failure of `test/authors` is attributed to a specific cause: it could occur when commits are performed using the GitHub web UI. This platform consistently logs the name as the GitHub username, while the email is logged in the format: `Geetika Vadali <123307246+geetHonve@users.noreply.github.com>`.

 In this case, the above line will need to be updated to `.mailmap` like

`Geetika Vadali <geetika047btcsai21@igdtuw.ac.in> Geetika Vadali <123307246+geetHonve@users.noreply.github.com>`

See here:
 https://docs.sympy.org/dev/contributing/new-contributors-guide/workflow-process.html#add-your-name-and-email-address-to-the-mailmap-file

Reply:
I'm sorry, I'm new to contributing. Can you tell what I can do to prevent this failure? Should I switch it to the username format as you mentioned?

Reply:
`Geetika Vadali <geetika047btcsai21@igdtuw.ac.in> Geetika Vadali <geetika.vadali4@gmail.com>`
`Geetika Vadali <geetika047btcsai21@igdtuw.ac.in> Geetika Vadali <123307246+geetHonve@users.noreply.github.com>`

Add these two lines in `.mailmap` file

Reply:
I added, I am getting a reordering issue.

Reply:
`python bin/mailmap_check.py` run this command.
See here 
https://docs.sympy.org/dev/contributing/new-contributors-guide/workflow-process.html#add-your-name-and-email-address-to-the-mailmap-file

Reply:
I am having troubles with a new name as Geetika V, i have no idea how it is added but it is causing ambiguity issue. can somebody suggest what to add

Reply:
`Geetika Vadali <geetika047btcsai21@igdtuw.ac.in> Geetika V <geetika047btcsai21@igdtuw.ac.in>`
You can do this.

Reply:
`python bin/mailmap_check.py` run this command.

Reply:
still the same ambiguity issue, been modelling against existing names and still

Reply:
If you run that command it should fix the problem. In CI it says:
```
The mailmap file was reordered
```
That command will reorder the file and then the check should pass in CI.

Reply:
`python bin/mailmap_check.py` command?

Reply:
resolved. pls check @maintainers

Reply:
Ideally I would rebase and arrange the changes into two commits:

1. Add contributor email to .mailmap file
2. Add tests

This pr has too many commits (many of them incorrect) while the code is of just 12 lines. But since this is a first time contribution, merging is also fine.

One thing I would always encourage you to do is give unique commit messages for each commit

Reply:
Understood. Apologies, I was kind of learning this ecosystem while correcting this pull request. I have done a certain squashing with respect to so many commits. If there's anything more you'd expect me to do, let me know. 
thanks for the guidance!

Reply:
+ test-pyodide (3) failing is not because of my changes right? it did pass yesterday night